config/routes.rb

```diff
@@ -78,7 +78,7 @@
   get "/find-licences/:slug/:authority_slug/:interaction", to: "licence_transaction#authority_interaction", as: "licence_transaction_authority_interaction"
 
   # Old style media previews (compatible with preview_url)
-  get "/media/:id/:filename/preview", to: "csv_preview#show", filename: /[^\/]+/
+  get "/media/:id/:filename/preview", to: "csv_preview_redirect#redirect", filename: /[^\/]+/
```

Comment:
nit pick! Please could we amend the comment, to lose `(compatible with preview_url)` as it wasn't and isn't!